src/StatsdClient/StatsdConfig.cs

```diff
@@ -140,9 +145,14 @@ public StatsdConfig()
         /// </summary>
         public ClientSideAggregationConfig ClientSideAggregation { get; set; } = new ClientSideAggregationConfig();
 
+        /// <summary>
+        /// Gets or sets the container id to tag the telemetry with.
+        /// </summary>
+        public string ContainerID { get; set; }
```

Comment:
We actually let callers set the container ID manually in other DSD client libraries? 🤔